src/pages/core/conventions/library-feature.mdx

```diff
@@ -0,0 +1,58 @@
+---
+tags: ["core", "conventions"]
+---
+
+import { Callout } from "nextra/components";
+
+# Library feature
+
+In the ecosystem, there is the convention to gate the entrypoints of your
+contract behind a compile-time feature. The feature is conventionally called
+`library`.
```

Comment:
Not really related to this PR, but I've been wondering for a while if we should adopt this for all code examples in the docs. I suspect many people will just copy and paste stuff from the docs. WDYT?

Reply:
Not sure. This is actually a good point though. But I'm also not sure how we would enforce this across the workspace.

Regex across the code samples for `#[entry_point]` and deny if it's hit?

Reply:
But I feel like this is a separate issue from this PR. I'll open a follow-up issue on the tracker.